apps/server/src/modules/task/controller/task.controller.ts

```diff
@@ -102,4 +103,50 @@ export class TaskController {
 
 		return result;
 	}
+
+	@Post()
+	async create(@Body() params: TaskCreateParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
+		this.FeatureTaskCardEnabled();
+
+		const taskWithSatusVo = await this.taskUc.create(currentUser.userId, TaskMapper.mapTaskCreateToDomain(params));
+
+		const response = TaskMapper.mapToResponse(taskWithSatusVo);
+		return response;
+	}
+
+	@Patch(':taskId')
+	async update(
+		@Param() urlParams: TaskUrlParams,
+		@Body() params: TaskUpdateParams,
+		@CurrentUser() currentUser: ICurrentUser
+	): Promise<TaskResponse> {
+		this.FeatureTaskCardEnabled();
+
+		const taskWithSatusVo = await this.taskUc.update(
+			currentUser.userId,
+			urlParams.taskId,
+			TaskMapper.mapTaskUpdateToDomain(params)
+		);
+
+		const response = TaskMapper.mapToResponse(taskWithSatusVo);
+
+		return response;
+	}
+
+	@Get(':taskId')
+	async findTask(@Param() urlParams: TaskUrlParams, @CurrentUser() currentUser: ICurrentUser): Promise<TaskResponse> {
+		this.FeatureTaskCardEnabled();
+
+		const taskWithSatusVo = await this.taskUc.find(currentUser.userId, urlParams.taskId);
+
+		const response = TaskMapper.mapToResponse(taskWithSatusVo);
+		return response;
+	}
+
+	private FeatureTaskCardEnabled() {
+		const enabled = Configuration.get('FEATURE_TASK_CARD_ENABLED') as boolean;
+		if (!enabled) {
+			throw new InternalServerErrorException('Feature not enabled');
```

Comment:
what about `ForbiddenException` instead?

Reply:
in this case, is not a permissions, is simply tryging to access an endpoint which is not enabled. So I would say 500 - because of server configuration - is ok